api/v1alpha1/access_control_validate.go

```diff
@@ -134,6 +134,21 @@ func GetUsersFromSpec(spec *AerospikeClusterSpec) map[string]AerospikeUserSpec {
 	return users
 }
 
+func isRoleQuotaParamValid(roleSpec AerospikeRoleSpec, aerospikeConfigSpec *AerospikeConfigSpec) error {
```

Comment:
The adjacent `isRoleSpecValid` returns `(bool, error)`. The first value never used though. If we return only an error the better name would be `validateRoleQuotaParam`.

Reply:
Agree, Done!